changelog.d/6888.feature

```diff
@@ -0,0 +1 @@
+The result of a user directory search can now be filtered via the spam checker.
```

Comment:
Should this be the issue number instead of the PR number in this case?

Reply:
no, we always put the PR number here. The changelog makes better reading when it links to what's changed rather than the reason it's changed.